lib/device/devicebuilder.js

```diff
@@ -716,7 +716,17 @@ const deviceBuilder = class DeviceBuilder {
    * @param {String} configuration.uri HTTP URI pointing to an image resource. JPG and PNG images are supported.
    * @param {String} configuration.size image size in the ui, either 'small' or 'large'. The small image has the size of a button while the large image is a square image using full width of the client.
    * @param {Function} controller A function which returns the address (URL) to the current image.
-   * @description Add an image to your custom element (for example to display the album cover of the current track)
+   * @description Add an image to your custom element (for example to display the album cover of the current track).
+   * To avoid downloading and resizing large images on the Brain use these general guidelines:
+   * - Large images: 480 x 480
+   * - Small: 100 x 100
+   * If you want to optimize for a specific target:
+   * - img url large: 454 x 454
+   * - img url small: 100 x 100
+   * - player: 480 x 480
+   * - list images (and mini player): 80 x 80
+   * - list tile images: 215 x 215
```

Comment:
This list images don't use the addImageUrl component, I think the sizes for list images would be more useful in the list builder where the list items are added.